.gitignore

```diff
@@ -30,3 +30,5 @@ app/assets/builds/
 .yardoc
 
 .devcontainer/output
+
+/config/master.key
```

Comment:
Smert

Reply:
it looks like it can be set via an environment variable as well - https://edgeguides.rubyonrails.org/security.html#custom-credentials

Reply:
Also, this was added automatically by whatever command was used to generate the key in the first place. Good for protecting me from myself....